src/content/docs/tutorial-diagnose-infra-bottlenecks/instrument-your-system.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: Collect data about your system infrastructure
+metaDescription: Capture data about your system by installing agents, then view that data in New Relic. 
+---
+
+Like the bridges and roads of a dense city, your system infrastructure needs regular inspection and maintenance. What processes affect CPU and disk usage? What app behaviors typically correlate with failing hosts? The performance of your infrastructure can affect the performance of your APIs, services, and apps. A database failure can prevent your server from handling data, or a memory leak can bring an otherwise performant app to an abrupt halt. 
```

Comment:
Is it worth saying why bridges and roads need to be inspected? to keep travel moving? to make sure people can get where they're going? Is it worth extending this metaphor into an active state that states a _why_?